src/backend/server.js

```diff
@@ -164,20 +161,17 @@ app.post("/toggle/on", async (req, res) => {
 })
 
 app.post("/massMessage", async (req, res) => {
-  console.log(req.body.userId)
-  console.log(keys.isAdminChatID)
-  if (req.body.userId != keys.isAdminChatID) {
+  if (req.body.userId !== keys.adminUserId) {
```

Comment:
Can you please stop changing this without double checking. The two variables here do not have the same type!

Reply:
Noted.